fourm/data/modality_transforms.py

```diff
@@ -217,8 +217,51 @@ def image_crop_and_resize(img: Image, crop_coords: Tuple, target_size: Tuple, re
         return img
 
 
-class RGBTransform(ImageTransform): # For RGB in raw format for 4M because it's passed in directly (and also training tokenizer)
+class VideoRGBTransform(AbstractTransform):
+    """
+    A video transform applied to a sequence of RGB images.
+    For now, I'm assuming the input to the load function points to a webdataset containing mp4 which contains the frames in the specified modality (e.g, RGB).
+    This format almost certainly is subject to change. TODO: figure out what the right format for this input should be (aka, how are we storing the raw videos?)
+
+    Output: a tensor of shape (num_frames, C, H, W) where C is the number of channels (3 for RGB). OR should this already be unrolled into something like (num_frames * C, H, W)?
+    """
+
+    # raise NotImplementedError("I'm not ")
+    def load(self, path):
```

Comment:
Note: webd formats are not loaded using this function. This is only applicable for non-webd data storage formats (Ali).

Reply:
https://github.com/swiss-ai/ml-4m/blob/4c2c9a56a9e2cd3e94316e766028c71bb6e248d8/fourm/data/unified_datasets.py#L370 - the pipeline for loading webd and running the preprocess/augment/postprocess. 

https://github.com/swiss-ai/ml-4m/blob/4c2c9a56a9e2cd3e94316e766028c71bb6e248d8/fourm/data/unified_datasets.py#L162 is the thing that actually loads the specific files from the webd tars

Reply:
All modalities should use webd and be compressed into webd/tar files.

Reply:
This means the load function will (probably) never be called for webd/our video-based modalities